### PR TITLE
DO NOT MERGE UNTIL 2.2.0 - Release documentation

### PR DESCRIPTION
**Pull request by susanmdouglas-aws**
### Description
This commit includes the changes to the documentation for release 2.2.0.  
 
### Issues Resolved
This commit includes the changes to the documentation for Babelfish release 2.2.0 (community release 14.5).  The documentation includes a blog post, release notes, updates to the escape hatches page, an updated unsupported features page (specific to 2.2.0), and the supporting artifacts that make the content display on the website.

### Check List
- [x] Commits are signed per the DCO using `--signoff`

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Review by 2jungkook — DISMISSED** [review]

**Review by geetanjalij — APPROVED** [review]

File: _posts/2022-09-8-babelfish-release-2-2-0.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+comments: true
+title: "Babelfish 2.2.0 is available"
+categories:
+- releases
+---
+
+We’re happy to announce the release of Babelfish 2.2.0, which includes community updates made to PostgreSQL 14.5. This release includes support for a number of features requested by our users:
+
+- Support for old-style function calls with `::` preceding the function name.
+- Support for more catalogs: assembly_types, numbered_procedures, triggers, spatial_index_tessellations, plan_guides, synonyms, events, trigger_events, fulltext_indexes, dm_hadr_cluster, xml_indexes, change_tracking_tables, key_constraints, database_filestream_options, filetable_system_defined_objects, hash_indexes, filegroups, master_files, assembly_modules, change_tracking_databases, database_recovery_status, fulltext_catalogs, fulltext_stoplists, fulltext_indexes, fulltext_index_columns, fulltext_languages, selective_xml_index_paths, spatial_indexes, filetables, registered_search_property_lists, syspolicy_configuration, syspolicy_system_health_state
```
Comment (suprio-amzn) [inline _posts/2022-09-8-babelfish-release-2-2-0.md:15]: Remove this line.

File: _includes/releases/babelfish-2.2.0.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+- Support for `CAST` function of numeric expressions to `DATETIME`.
+- Support for `@@LANGUAGE` variable with the constant value of `us_english`.
+- Support for the old-style function calls with `::` preceding the function name.
+- Support for the `sp_helpsrvrolemember` stored procedure.
+- Support for the `msdb.dbo.fn_syspolicy_is_automation_enabled system` function.
+- Support for more catalogs: Support for more catalogs: `assembly_types`, `numbered_procedures`, `triggers`, `spatial_index_tessellations`, `plan_guides`, `synonyms`, `events`, `trigger_events`, `fulltext_indexes`, `dm_hadr_cluster`, `xml_indexes`, `change_tracking_tables`, `key_constraints`, `database_filestream_options`, `filetable_system_defined_objects`, `hash_indexes`, `filegroups`, `master_files`, `assembly_modules`, `change_tracking_databases`, `database_recovery_status`, `fulltext_catalogs`, `fulltext_stoplists`, `fulltext_indexes`, `fulltext_index_columns`, `fulltext_languages`, `selective_xml_index_paths`, `spatial_indexes`, `filetables`, `registered_search_property_lists`, `syspolicy_configuration`, `syspolicy_system_health_state`.
```
Comment (suprio-amzn) [inline _includes/releases/babelfish-2.2.0.md:38]: Repetition.

File: _posts/2022-09-8-babelfish-release-2-2-0.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    - Bill Ramos
+comments: true
+title: "Babelfish 2.2.0 is available"
+categories:
+- releases
+---
+
+We’re happy to announce the release of Babelfish 2.2.0, which includes community updates made to PostgreSQL 14.5. This release includes support for a number of features requested by our users:
+
+- Support for old-style function calls with `::` preceding the function name.
```
Comment (suprio-amzn) [inline _posts/2022-09-8-babelfish-release-2-2-0.md:14]: Add the other new features here:

- Support for data migration using the BCP client and the bcp utility
- Support for cross-database stored procedure execution
- Support for `CROSS APPLY` and `OUTER APPLY` (lateral join)
- Support for additional catalog objects

File: _posts/2022-09-8-babelfish-release-2-2-0.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+- Support for the `babelfishpg_tsql.escape_hatch_showplan_all` [escape hatch](https://babelfishpg.org/docs/usage/escape_hatches/). When set to `ignore`, the `SET SHOWPLAN_ALL` and `SET STATISTICS PROFILE` will behave like `SET BABELFISH_SHOWPLAN_ALL` and `SET BABELFISH_STATISTICS PROFILE`; when set to `strict`, `SET SHOWPLAN_ALL` and `SET STATISTICS PROFILE` are silently ignored. Note that the query plan information displayed is PostgreSQL-style, rather than SQL Server-style. For more information, see [Reviewing a Query Plan](https://babelfishpg.org/docs/usage/query_planning/).
```
Comment (suprio-amzn) [inline _posts/2022-09-8-babelfish-release-2-2-0.md:16]: Remove this line.

**Review by suprio-amzn — CHANGES_REQUESTED** [review]

File: _posts/2022-09-8-babelfish-release-2-2-0.md (hunk 3 above)
Comment (susanmdouglas-aws) [reply _posts/2022-09-8-babelfish-release-2-2-0.md:14]: Added...

File: _posts/2022-09-8-babelfish-release-2-2-0.md (hunk 4 above)
Comment (susanmdouglas-aws) [reply _posts/2022-09-8-babelfish-release-2-2-0.md:16]: Removed...

**Review by kuntalghosh — APPROVED** [review]

File: _includes/releases/babelfish-2.2.0.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+- Fixed case-sensitivity issue with arguments to the functions `USER_ID` and `SUSER_ID`.
+- Fixed an issue where triggers were allowed to be created on Babelfish temporary tables.
+- Fixed several performance issues with Import-Export wizard.
+- Support for multi-byte client encodings other than `UTF-16` for `VARCHAR(n)`.
+- Fixed the system compatibility view `sys.sysprocesses` to show the correct value for hostname provided by the client connection.
+- Fixed case sensitivity issue with `Polish_CI_AS` collation.
+- Fixed the `@@DBTS` function so that value of `@@DBTS` correctly returns the current transaction id after each DML statement even when used within a transaction.
+- Improved performance for queries that refer to the functions `SCOPE_IDENTITY` and `@@IDENTITY`.
+- Support added for collations `Japanese_CS_AS`, `Japanese_CI_AI` and `Japanese_CI_AS` for `fn_helpcollations`.
+- `@@SERVERNAME` and `SERVERPROPERTY('ServerName')` now return the name of the Babelfish instance as specified by the user when the instance is created. This value is also returned by the newly supported properties `SERVERPROPERTY('MachineName')` and `SERVERPROPERTY('InstanceName')`.
```
Comment (suprio-amzn) [inline _includes/releases/babelfish-2.2.0.md:77]: Remove this line.

**Review by suprio-amzn — CHANGES_REQUESTED** [review]

File: _includes/releases/babelfish-2.2.0.md (hunk 5 above)
Comment (susanmdouglas-aws) [reply _includes/releases/babelfish-2.2.0.md:77]: Gone

**Review by shardgupta — DISMISSED** [review]

**Review by kuntalghosh — APPROVED** [review]

**Comment by susanmdouglas-aws** [comment]
I've pushed this PR at the command line, so am manually closing the PR...

**Pull request closed without merging**